packages/ihe-gateway/docker-compose.yml

```diff
@@ -5,31 +5,34 @@ services:
       context: .
       dockerfile: Dockerfile
       args:
-        ARTIFACT: "${ARTIFACT_URL}"
-        STOREPASS: "${STOREPASS}"
-        KEYSTOREPASS: "${KEYSTOREPASS}"
-        KEYSTORENAME: "${KEYSTORENAME}"
-        ZULUKEY: "${ZULUKEY}"
+        # Update this if you're trying to run it on a different platform
+        PLATFORM: "linux/arm64"
+        ARTIFACT: "${IHE_GW_ARTIFACT_URL}"
+        KEYSTORENAME: "${IHE_GW_KEYSTORENAME}"
+        ZULUKEY: "${IHE_GW_ZULUKEY}"
       x-bake:
         platforms:
-          - linux/amd64    
+          - linux/amd64
           - linux/arm64
```

Comment:
why do we have both `amd64` and `arm64` listed if above we only specify `linux/arm64` ?

Reply:
Just kept what we had. I assumed it was like that to indicate to Docker those are the accepted platforms and the order of preference (so I reordered).

Reply:
yeah for sure, that makes sense - if that's what we already have it's a super low prio, was mainly wondering as I don't understand the "why"